v1.3.1.md

```diff
@@ -58,6 +58,9 @@ Harvester only allows upgrades from supported versions. For more information abo
 > [!NOTE]
 > You can upgrade to v1.3.1 from both v1.3.0 and v1.2.2. If you are using an earlier version, you must upgrade to v1.2.2 before upgrading to v1.3.1.
 
+> [!WARNING]
+> Please check https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1 for known issues. There are some reports from the community about issue [#6041](https://github.com/harvester/harvester/issues/6041). Please check the [Workaround](https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1#1-upgrade-stuck-after-the-first-node-upgraded) for how to resume the upgrade.
```

Comment:
```suggestion
> [!IMPORTANT]
> You can upgrade to v1.3.1 from both v1.3.0 and v1.2.2. If you are using an earlier version, you must upgrade to v1.2.2 before upgrading to v1.3.1.
> 
> The [documentation](https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1) contains information about upgrade-related issues. If your cluster is affected by [issue #6041](https://github.com/harvester/harvester/issues/6041), check the suggested [workaround](https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1#1-upgrade-stuck-after-the-first-node-upgraded) in the documentation.
```

Reply:
I suggest combining the notes to avoid cluttering the release notes. I think "Important" is more appropriate for the new text.

Reply:
Thank you! Merge the change.